FontAtlas.go

```diff
@@ -39,9 +39,14 @@ func (atlas FontAtlas) GlyphRangesJapanese() GlyphRanges {
 	return GlyphRanges(C.iggGetGlyphRangesJapanese(atlas.handle()))
 }
 
-// GlyphRangesChinese describes Default + Japanese + full set of about 21000 CJK Unified Ideographs.
-func (atlas FontAtlas) GlyphRangesChinese() GlyphRanges {
-	return GlyphRanges(C.iggGetGlyphRangesChinese(atlas.handle()))
+// GlyphRangesChineseFull describes Default + Japanese + full set of about 21000 CJK Unified Ideographs.
+func (atlas FontAtlas) GlyphRangesChineseFull() GlyphRanges {
+	return GlyphRanges(C.iggGetGlyphRangesChineseFull(atlas.handle()))
+}
+
+// GlyphRangesChineseSimplifiedCommon describes Default + Japanese + full set of about 21000 CJK Unified Ideographs.
```

Comment:
What is the difference between `GlyphRangesChineseFull` and `GlyphRangesChineseSimplifiedCommon` if their comments are the same?

Reply:
`GlyphRangesChinese` was renamed to `GlyphRangesChineseFull` in imgui 1.62. I've restored the original function for compatibility reasons, but added `GlyphRangesChineseFull`, and updated `GlyphRangesChinese` to instead call it. 

I wasn't 100% sure on that change, as it introduces an additional function call, but wanted to keep code duplication low and function naming conventions consistent with imgui's.

Reply:
Apologies, fell into the copy-paste trap there. Comments for both have been updated to keep in line with imgui's comments